crates/egui_extras/src/loaders/image_loader.rs

```diff
@@ -90,19 +150,14 @@ impl ImageLoader for ImageCrateLoader {
                             });
                         }
                     }
-
-                    if bytes.starts_with(b"version https://git-lfs") {
-                        return Err(LoadError::FormatNotSupported {
-                            detected_format: Some("git-lfs".to_owned()),
-                        });
+                    #[cfg(not(target_arch = "wasm32"))]
+                    {
+                        load_image(ctx, uri, &self.cache, bytes)
+                    }
+                    #[cfg(target_arch = "wasm32")]
+                    {
+                        load_image(ctx, uri, &self.cache, &bytes)
```

Comment:
just use the exact same file signature instead:

```suggestion
                        load_image(ctx, uri, &self.cache, bytes)
```